python/foglamp/services/core/api/common.py

```diff
@@ -86,7 +86,7 @@ def services_health_litmus_test():
     status_color = services_health_litmus_test()
     safe_mode = True if server.Server.running_in_safe_mode else False
 
-    return web.json_response({'uptime': since_started,
+    return web.json_response({'uptime': int(since_started),
```

Comment:
thanks for this (I will update jira for this reference too)

Also please fix for `python.foglamp.services.core.server.Server#ping`